apollos-church-api/src/data/ContentItem.js

```diff
@@ -185,15 +185,15 @@ class dataSource extends ContentItem.dataSource {
     const { Feature, Matrix, BinaryFiles } = this.context.dataSources;
 
     // scripture ref
-    const reference = item.attributeValues.bookoftheBible?.value;
-    if (reference) {
-      features.push(
-        Feature.createScriptureFeature({
-          reference,
-          id: `${item.attributeValues.bookoftheBible.id}`,
-        })
-      );
-    }
+    // const reference = item.attributeValues.bookoftheBible?.value;
```

Comment:
If we don't need it, we should just delete it

Reply:
Oh okay!